travis/check_tags.py

```diff
@@ -0,0 +1,130 @@
+                    'seven': '7', 'eight': '8', 'nine': '9'}
+VERSION_TAGS = [':zero:', ':one:', ':two:', ':three:', ':four:', ':five:', ':six:', ':seven:', ':eight:', ':nine:']
+REQUIREMENTS_TAGS_OF_VERSION = [':x:', ':arrow_up:', ':arrow_down:', ':sos:', ':tada:']
+
+
+def get_errors_msgs_commits(travis_repo_slug, travis_pull_request_number, travis_branch, version, token):
+    symbol_in_branch = re.search(r'-', str(travis_branch))
+
+    #GET /repos/:owner/:repo/pulls/:pull_number/commits
+    #See API Github: https://developer.github.com/v3/repos/commits/#list-commits-on-a-repository
+    real_errors = {}
+    if not travis_pull_request_number:
+        return real_errors
+    # GET / repos /: owner /:repo / commits
+    url_request = 'https://api.github.com/repos/%s/pulls/%s/commits' % (str(travis_repo_slug), str(travis_pull_request_number))
+    if token:
+        commits = requests.get(url_request, headers={'Authorization': token}).json()
+    else:
+        commits = requests.get(url_request).json()
+    for commit in commits:
+        commit = commit.get('commit').get('message')
+        print('Commit: %s' % commit)
+        if commit:
+            errors_commit = handler_commit(commit, symbol_in_branch, version)
+            real_errors.update(errors_commit)
+    return real_errors
+
+
+def handler_commit(commit, symbol_in_branch, version):
+    errors_commit = {}
+    # looks tags starting at the beginning of the line and until first whitespace
+    match_tags_commit = re.search(r'^(:[^\s]+:)', commit).group(1)
+    if not match_tags_commit:
+        error = {commit: 'There are no tags in the commit!'}
+        errors_commit.update(error)
+        return errors_commit
+    # list of tags from match_tags_commit
+    list_tags = re.findall(r'(:\w+:)', match_tags_commit)
+    # list of tags that should not be in the commit
+    extra_tags = [i for i in list_tags if i not in DEVELOPMENT_TAGS + RELEASE_TAGS + VERSION_TAGS]
+    if extra_tags != []:
+        error = {commit: 'There should not be such tags in the commit!'}
+        errors_commit.update(error)
+        return errors_commit
+    # lists of Development tag and Release tag in commit
+    dev_tag = list(set(list_tags) & set(DEVELOPMENT_TAGS))
+    release_tag = list(set(list_tags) & set(RELEASE_TAGS))
+    if symbol_in_branch:
+        errors_dev = check_dev_branch_tags(dev_tag, commit, list_tags)
+        errors_commit.update(errors_dev)
+    else:
+        errors_stable = check_stable_branch_tags(release_tag, commit, list_tags)
+        errors_commit.update(errors_stable)
+
+    if any(tag in REQUIREMENTS_TAGS_OF_VERSION for tag in list_tags):
+        errors_version = check_version_tags(list_tags, commit, version)
+        errors_commit.update(errors_version)
+    return errors_commit
+
+
+def check_version_tags(list_tags, commit, version):
+    errors_version = {}
+    if not any(tag in VERSION_TAGS for tag in list_tags):
```

Comment:
make variable ``version_tags``

Reply:
And don't use ``not any``. You can always replace it with ``all(`` (though with a different condition)